configure-git/action.yml

```diff
@@ -5,4 +5,5 @@ runs:
   using: composite
   steps:
     - shell: bash
+      working-directory: ${{ github.action_path }}
```

Comment:
I forgot to include this in #25. Explanation: when GH runs an action, PWD is still the checked-out _workflow_ unless you include this